pkg/cloud/amazon/aws_kms_test.go

```diff
@@ -98,8 +98,7 @@ func TestEncryptDecryptAWS(t *testing.T) {
 		// https://docs.aws.com/cli/latest/userguide/cli-configure-role.html
 		// We only run this test if default role exists.
 		ctx := context.Background()
-		cfg, err := config.LoadDefaultConfig(ctx,
-			config.WithSharedConfigProfile(config.DefaultSharedConfigProfile))
```

Comment:
Our `implicit` tests shouldn't be enforcing that a `DefaultSharedConfigProfile` is loaded